server/src/main/kotlin/org/javacs/kt/symbols/Symbols.kt

```diff
@@ -33,10 +34,10 @@ private fun doDocumentSymbols(element: PsiElement): List<DocumentSymbol> {
     } ?: children
 }
 
-fun workspaceSymbols(query: String, sp: SourcePath): List<WorkspaceSymbol> =
+fun workspaceSymbols(locationRequired: Boolean, query: String, sp: SourcePath): List<WorkspaceSymbol> =
```

Comment:
Style nit: I would pass the `locationRequired` parameter last, to maintain the ordering of decreasing significance.